acceptance-tests/src/test/resources/uk/gov/di/test/features/api/account-management/account_management.feature

```diff
@@ -8,4 +8,3 @@ Feature: Account Management
     Then the system sends an OTP to "07700900111"
     When the User provides the correct otp
     Then the User's Phone Number is updated to "07700900111"
-
```

Comment:
This should be rebased into the previous commit

Reply:
Hi @whi-tw this is actioned. please review